src/OpenLoco/src/Environment.cpp

```diff
@@ -247,10 +250,31 @@ namespace OpenLoco::Environment
         }
         setDirectory(_pathSavesSinglePlayer, saveDirectory);
         setDirectory(_pathSavesTwoPlayer, saveDirectory);
-        autoCreateDirectory(saveDirectory);
 
+        // Figure out what landscape directory to default to.
+        // NB: vanilla routines do not use std::filesystem yet, so the trailing slash is still needed.
+        auto landscapeDirectory = getPathNoWarning(PathId::landscape) / "";
+        autoCreateDirectory(landscapeDirectory);
+
+        auto& configLastLandscapePath = Config::get().lastLandscapePath;
+        if (!configLastLandscapePath.empty())
+        {
+            // Getting the directory can fail if config is bad.
+            try
+            {
+                auto directory = fs::u8path(configLastLandscapePath);
+                if (fs::is_directory(directory))
+                {
+                    landscapeDirectory = directory;
+                }
+            }
+            catch (std::system_error&)
+            {
```

Comment:
should we show the user an error message here?

Reply:
i just saw its the same code as above for save files - should we instead move these bits of code into a common method? they're otherwise identical

Reply:
In this case, the `landscapeDirectory` variable will just remain the default. I believe we can keep quiet here.